content/en/synthetics/api_tests/tcp_tests.md

```diff
@@ -19,12 +19,19 @@ further_reading:
 
 ## Overview
 
-TCP tests allow you to easily **monitor that low-level TCP connections can be established on the ports of given hosts**, ensuring availability of a variety of key services: e.g., `SSH` (22), `SMTP` (25), `DNS` (53), VPN over `HTTPS` (443), and any custom services living on other ports. Thanks to built-in response time data, you can keep track of the performance of your network applications and be alerted in case of unexpected slowness.
-TCP tests can run from [managed][1] and [private locations][2] depending on whether you want to launch your TCP connections from **outside or inside your network**.
+TCP tests allow you to easily monitor whether or not low-level TCP connections can be established on the ports of given hosts, ensuring the availability of several key services such as `SSH` (22), `SMTP` (25), `DNS` (53), VPN over `HTTPS` (443), and any custom services living on other ports. With built-in response time data, you can keep track of the performance of your network applications and be alerted in case of unexpected slowness. 
```

Comment:
"With built-in response time data" what do you mean by that?

Reply:
I'm not sure exactly! Should we remove it? @HugoPct

Reply:
Indeed you can keep it.

+ I'm not that fan of "whether or not". A bit negative. 
Could it be replace by "if" ? Or something where we do not offer the "yes or not" schema